recipes/tinygltf/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "tinygltf" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/syoyo/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 5d85bd556b60b1b69527189293cfa4902957d67fabb8582b6532f23a5ef27ec1
+
+build:
```

Comment:
Are there any ABI concerns to keep track of? i.e. could structures from this be passed between downstream packages?

Reply:
I don't believe so.